packages/web-components/src/avatar/avatar.options.ts

```diff
@@ -0,0 +1,124 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * The Avatar "active" state
+ */
+export const AvatarActive = {
+  active: 'active',
+  inactive: 'inactive',
+} as const;
+
+/**
+ * The types of Avatar active state
+ */
+export type AvatarActive = ValuesOf<typeof AvatarActive>;
+
+/**
+ * The Avatar Shape
+ */
+export const AvatarShape = {
+  circular: 'circular',
+  square: 'square',
+} as const;
+
+/**
+ * The types of Avatar Shape
+ */
+export type AvatarShape = ValuesOf<typeof AvatarShape>;
+
+/**
+ * The Avatar Appearance when "active"
+ */
+export const AvatarAppearance = {
+  ring: 'ring',
+  shadow: 'shadow',
+  ringShadow: 'ring-shadow',
+} as const;
+
+/**
+ * The appearance when "active"
+ */
+export type AvatarAppearance = ValuesOf<typeof AvatarAppearance>;
+
+/**
+ * A specific named color for the Avatar
+ */
+export const AvatarNamedColor = {
+  darkRed: 'dark-red',
+  cranberry: 'cranberry',
+  red: 'red',
+  pumpkin: 'pumpkin',
+  peach: 'peach',
+  marigold: 'marigold',
+  gold: 'gold',
+  brass: 'brass',
+  brown: 'brown',
+  forest: 'forest',
+  seafoam: 'seafoam',
+  darkGreen: 'dark-green',
+  lighTeal: 'light-teal',
+  teal: 'teal',
+  steel: 'steel',
+  blue: 'blue',
+  royalBlue: 'royal-blue',
+  cornflower: 'cornflower',
+  navy: 'navy',
+  lavender: 'lavender',
+  purple: 'purple',
+  grape: 'grape',
+  lilac: 'lilac',
+  pink: 'pink',
+  magenta: 'magenta',
+  plum: 'plum',
+  beige: 'beige',
+  mink: 'mink',
+  platinum: 'platinum',
+  anchor: 'anchor',
+} as const;
+
+/**
+ * An avatar can be one of named colors
+ * @public
+ */
+export type AvatarNamedColor = ValuesOf<typeof AvatarNamedColor>;
+
+/**
+ * Supported Avatar colors
+ */
+export const AvatarColor = {
+  neutral: 'neutral',
```

Comment:
Do this repeat itself in any other fluent components?

Reply:
I don't think we should start with abstractions if that's the question